src/com/article-changer-v3.js

```diff
@@ -13,8 +13,16 @@
 			ctn.css({
 				minHeight: current.height() + 'px'
 			});
+			App.mediator.notify('articleChanger.leaving', {
+				item: current,
```

Comment:
we should drop `item` no one uses it and we should all stick to one.

Reply:
@fhamon so only article ?

Reply:
Yep. We use `article` everywhere.

Reply:
@fhamon ok